tox.ini

```diff
@@ -123,3 +123,13 @@ pass_env =
 commands =
     poetry install --with integration
     poetry run pytest -vv --no-header --tb native --log-cli-level=INFO -s {posargs} {[vars]tests_path}/integration/test_tls.py
+
+[testenv:integration-ha]
```

Comment:
As mentioned in the other PR, now that we may end-up with many tests, we could actually make this a bit more parametric, as we do in data platform libs

https://github.com/canonical/data-integrator/blob/main/tox.ini#L64

Reply:
While it's a good feature to have, it's not clear to me WHAT we want to parameterize at this stage.
We could do it to reduce LOC in `tox.ini`, but would prefer to leave that out of this PR as we're separating out `'ha'` tests from the others. If we were to use params, it would bundle all the other tests together, which isn't part of this PR.

Reply:
Totally fine to leave this out-of-scope. It was more of a suggestion, as I notifced that the tox.ini is become increasingly complex, with some copy and paste